helm/sealed-secrets/templates/servicemonitor.yaml

```
@@ -12,6 +12,7 @@ metadata:
     {{- if .Values.metrics.serviceMonitor.labels }}
     {{- include "sealed-secrets.render" ( dict "value" .Values.metrics.serviceMonitor.labels "context" $) | nindent 4 }}
     {{- end }}
+  annotations: {{- toYaml .Values.metrics.serviceMonitor.annotations | nindent 4 }}
```

Let's use the "sealed-secrets.render" that named template that allows you using templates in the annotations. Also, let's avoid render them unless they're set:
```suggestion
  {{- if .Values.metrics.serviceMonitor.annotations }}
  annotations: {{- include "sealed-secrets.render" ( dict "value" .Values.metrics.serviceMonitor.annotations "context" $) | nindent 4 }}
  {{- end }}
```

---

Done.

Thanks @juan131 and happy new year!

Please review again :pray: